src/security.ts

```diff
@@ -61,10 +63,11 @@ export const acmCertificateExpiration: StackValidationPolicy = {
         },
         validateStack: validateStackResourcesOfType(aws.acm.Certificate, async (acmCertificates, args, reportViolation) => {
             const { maxDaysUntilExpiration } =  args.getConfig<AcmCertificateExpirationArgs>();
-            const acm = new AWS.ACM();
+            // Need to pass in aws region for acm.
+            const acm = new AWS.ACM({region: awsConfigRegion});
```

Comment:
This likely won't work for stacks that use multiple regions with different AWS provider instances.

To support that, I think it'd look something like the following (I haven't tried it), though, I'm not sure if `region` will be available as a property on the provider:

```typescript
// Fetch the full ACM certificate using the AWS SDK to get its expiration date.
for (const certInStack of acmCertificates) {
    const region = certInStack.provider?.props.region ?? aws.config.region;
    const acm = new AWS.ACM({region: region});
    const describeCertResp = await acm.describeCertificate({ CertificateArn: certInStack.id }).promise();
    ...
}
```

Note: The use of `?.` and `??` requires upgrading typescript to at least 3.7.0.